src/alias.rs

```diff
@@ -48,6 +48,11 @@ impl Alias {
         let alias = chunks.next()?;
         let mime_type = chunks.next()?;
 
+        // Consume the leftovers, if any
+        if chunks.next().is_some() {
+            return None;
```

Comment:
Same considerations for `Subclass` applies here:

 - the spec is silent about additional tokens
 - the reference implementation does not care
 - we can be more strict without worrying too much about it